app/views/case_workers/claims/_claims.html.haml

```diff
@@ -28,14 +28,14 @@
           %td
             = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link'}
           %td
-            = claim.advocate.user.name
+            = claim.advocate_name
```

Comment:
The advocate_name can actually be:
```= claim.advocate.name```
as advocate delegates `name` to user anyway. In which case is there any benefit to advocate_name?